src/library.ver

```diff
@@ -1,11 +1,15 @@
 VERS_1.0 {
     local:
-        _Znwm;
-        _ZnwmRKSt9nothrow_t;
-        _ZdlPv;
-        _ZdlPvRKSt9nothrow_t;
-        _Znam;
-        _ZnamRKSt9nothrow_t;
-        _ZdaPv;
-        _ZdaPvRKSt9nothrow_t;
+        _Znwm;                 # operator new(unsigned long);
+        _ZnwmRKSt9nothrow_t;   # operator new(unsigned long, std::nothrow_t const&);
+        _ZdlPv;                # operator delete(void*);
+        _ZdlPvRKSt9nothrow_t;  # operator delete(void*, std::nothrow_t const&);
+        _Znam;                 # operator new[](unsigned long);
+        _ZnamRKSt9nothrow_t;   # operator new[](unsigned long, std::nothrow_t const&);
+        _ZdaPv;                # operator delete[](void*);
+        _ZdaPvRKSt9nothrow_t;  # operator delete[](void*, std::nothrow_t const&);
+
+        extern "C++" {
```

Comment:
Can we add a comment here explaining why we need to hide these?

Reply:
Added